database/seeders/RoleAndPermissionSeeder.php

```diff
@@ -47,6 +47,8 @@ public function run()
             Permission::findOrCreate('view_telescope'),
             Permission::findOrCreate('view_horizon'),
             Permission::findOrCreate('view_pulse'),
+	    Permission::findOrCreate('view_any_user'),
```

Comment:
this should just be `view_admin_panel` to give access to the filament admin panel, the user view and user update should be based on the user policy.